superbench/monitor/monitor.py

```diff
@@ -215,13 +238,21 @@ def __get_process_cpu_ticks(self):
         system_time = 0
         try:
             with open(self._cpu_file, 'r') as f:
-                for line in f:
-                    items = line.split()
-                    if items[0] == 'user':
-                        user_time = int(items[1])
-                    elif items[1] == 'system':
-                        system_time = int(items[1])
-                return user_time + system_time
+                if self.__cgroup == 1:
+                    for line in f:
+                        items = line.split()
+                        if items[0] == 'user':
+                            user_time = int(items[1])
+                        elif items[1] == 'system':
+                            system_time = int(items[1])
+                else:
+                    for line in f:
+                        items = line.split()
+                        if items[0] == 'user_usec':
+                            user_time = int(items[1]) / 10000
+                        elif items[1] == 'system_usec':
+                            system_time = int(items[1]) / 10000
+            return user_time + system_time
```

Comment:
read usage_usec directly?

Reply:
usage = (end_clock - begin_clock) / duration.
The duration can be controlled by monitor itself.